src/control/server/storage/bdev/backend_json.go

```diff
@@ -251,12 +251,17 @@ func newSpdkConfig(log logging.Logger, req *storage.BdevWriteConfigRequest) (*Sp
 				if bsc.Method == SpdkBdevNvmeSetHotplug {
 					bsc.Params = NvmeSetHotplugParams{
 						Enable:     true,
-						PeriodUsec: HotplugPeriod,
+						PeriodUsec: uint64(hotplugPeriod.Microseconds()),
```

Comment:
Whoa. I forgot that the `time.Duration` type had these methods! Even more convenient. Nice catch.